tools/clang/unittests/HLSL/AllocatorTest.cpp

```diff
@@ -663,7 +665,7 @@ TEST_F(AllocatorTest, GapFilling) {
 
 TEST_F(AllocatorTest, Allocate) {
   WEX::TestExecution::SetVerifyOutput verifySettings(WEX::TestExecution::VerifyOutputSettings::LogOnlyFailures);
-  for (auto &&scenario : m_Scenarios) {
+  for (auto &scenario : m_Scenarios) {
```

Comment:
You're under no obligation to fix this entire file, but there are a number of other areas where this is done unnecessarily.